docs/index.md

```diff
@@ -43,6 +73,26 @@ core_lib:
         protocol: sqlite
   ...
 ```
+#### `your_core_lib.yaml` Explained:
+<b>your_core_lib.yaml is the setting for your entire Core-Lib library. The above example will show how to configure core-lib to connect to a database using [SQLAlchemy](https://docs.sqlalchemy.org/en/20/){:target="_blank"}:</b>
+- <b>log_queries:</b> Default `True`, [echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag){:target="_blank"} is set to true. Set to false, SQL queries won’t be logged.
```

Comment:
- <b>log_queries:</b> Default `True`, Boolean indicating if to log SQL queries. see 
[echo flag](https://docs.sqlalchemy.org/en/20/core/engines.html#more-on-the-echo-flag){:target="_blank"} 


Are you sure the default it True? I think it's False